bin/wrap-xml-feed.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# If there is no xml header in the given file it will prepend a header,
+# and append a footer to make it legal xml feed.
+
+head -1 $1 | grep "xml version"
```

Comment:
This will output the content the first line if "xml version" is matched. This means the first line is outputted two times as the entire file is cat'ed at line 13.

Reply:
Thanks, fixed